Pal/regression/Hex.c

```diff
@@ -7,10 +7,10 @@
 #include "hex.h"
 
 int main() {
-  char buf[17];
-  uint32_t x = 0xefbeadde;
-  pal_printf("Hex test 1 is %s\n", hex2str(&x, buf, 17));
-  uint64_t y = 0xcdcdcdcdcdcdcdcd;
-  pal_printf("Hex test 2 is %s\n", hex2str(&y, buf, 17));
-  return 0;
+    char x[] = {0xde, 0xad, 0xbe, 0xef};
+    uint64_t y = 0xcdcdcdcdcdcdcdcd;
```

Comment:
Could you please fix `y` also? ;) It will work, but it's still an UB according to the standard and may fail someday (e.g. after a compiler update).